dbt_worker/tasks.py

```diff
@@ -107,19 +108,18 @@ def _invoke_runner(
         # artifacts may write to incorrect locations
         if project_dir:
             os.chdir(project_dir)
-        # TODO: Make sure callback works
-        dbt = dbtRunner(callbacks=[log_event_to_console])
+        dbt = dbtRunner()
         dbt.invoke(command)
+        logger.info(f"Task with id: {task_id} has completed successfully")
```

Comment:
Maybe just "completed"?

"successfully" might confuse us later and think the dbtRunner itself was successful?